### DISCUSSION TITLE
$_SERVER['REQUEST_URI'] not set and phpinfo() throws error

### DISCUSSION THREAD

**Discussion by MiguelDebruyne** [yandeu/five-server#130] [Help / Questions]
I did the installation, installed XAMPP for the first time, I've done no edits to config files except in VSC I added `/usr/local/bin/php` to the PHP executable path. No .htaccess in my project. It works, however, $_SERVER['REQUEST_URI'] is not set, when I looked further trying to debug I saw phpinfo() just throws an error `PHP Fatal error:  Uncaught Error: Call to undefined function php_info()`. Any idea why that is?

Edit: I've just come back from a long hiatus in programming, someone asked me to do php_info() while it should have been phpinfo(), which does work. I typed it here correctly, funnily enough.

Working on MacOs Big Sur.

**Comment by yandeu** [comment]
Five-Server does not provide a full php server.
Please see https://github.com/yandeu/five-server-with-php

**Comment by MiguelDebruyne** [comment]
My apologies, I should have seen that. I thought this was so basic that it's impossible for anyone to develop a website in php without that and my knowledge is to small to have seen that this is a php server issue. Feel free to delete.
